js/shared.js

```diff
@@ -374,3 +376,160 @@ async function findMatchingIncipits(incipit, titleStart) {
             return Promise.resolve([])
         })
 }
+
+/**
+ * @param source A String that is either a text body or a URI to a text resource.
+ */ 
+async function getAllWitnessesOfSource(source){
+    const linesLoaded = document.querySelector("tpen-line-selector").hasAttribute("tpen-lines-loaded") ? true : false
```

Comment:
This assignment is redundant. The `.hasAttribute()` returns Boolean

Reply:
oo yeah missed that.  I'll make this change.